client/ayon_deadline/lib.py

```diff
@@ -8,3 +10,36 @@
     "karma_rop", "vray_rop", "redshift_rop",
     "renderFarm", "usdrender", "publish.hou"
 ]
+
+# Constant defining where we store job environment variables on instance or
+# context data
+JOB_ENV_DATA_KEY: str = "farmJobEnv"
```

Comment:
I would add this constant to `__init__` of `ayon_deadline`.

Reply:
As you've said in the other comments - this may change when supporting e.g. multiple jobs, etc. - so I'm a bit worried to do anything that makes it feel like "the official API" for now. Once 'job env vars' need to be accessible by other addons this should actually even start moving to `ayon_core` because the env vars should be transferrable to other farm renderers as well.

Reply:
Make sense, can you also create PR adding the constant to ayon-core after this PR?

Reply:
I'd like to track it as an issue for now - because it may be that we'll need to redefine the var or its usage to nicely support multiple jobs on one instance with env vars of its own.

Reply:
With this PR, we already have defined something, that might be deprecated in future, but it is already defined...